### DISCUSSION TITLE
Lerna 7 & slow performance on Github Actions / Overwhelming Intel Mac laptops

### DISCUSSION THREAD

**Discussion by notethis** [lerna/lerna#3811] [Q&A]
Hi there!

I've been investigating upgrading my monorepo to Lerna 7 and enabling caching.  Locally on my machine (an M1 Mac), it works fantastic.  However, my colleagues who are still on Intel Silicon Macs are noticing the amount of node processes spinning up during our test phase is overwhelming their machine (colloquially, "melting my laptop").  On our Github Actions CICD step, the test stage went from about 5 minutes to 19 minutes.  

I've tried a few things to determine what is going on, ex: limiting steps that are cached to only the parent `build` steps, not `build:less`, etc.  And I've run the NX command to generate a `profile.json` file but that doesn't seem to provide any clues.  

Setting a `--concurrency` flag did not seem to help either

My repo has about 23 packages in it.  Has anyone encountered this before?  Any suggestions on what to try out next?

**Comment by fahslaj** [comment]
> Setting a --concurrency flag did not seem to help either
Are you using `lerna run` with the `--parallel` flag? If so, that will override `--concurrency`.

> the test stage went from about 5 minutes to 19 minutes
What test runner are you using? Are you able to determine if the tests themselves are taking longer, or if it's something else before/after the actual tests?
